cmd/orgs.go

```diff
@@ -215,6 +233,148 @@ func orgsRemove(ctx *cli.Context) error {
 	return nil
 }
 
+func orgsMembersListCmd(ctx *cli.Context) error {
+
+	// Use "member" team name to bypass specific teams
+	teamName := "member"
+
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		return err
+	}
+
+	client := api.NewClient(cfg)
+	c := context.Background()
+
+	var getMembers sync.WaitGroup
+	getMembers.Add(2)
+
+	var org *envelope.Org
```

Comment:
A small stylistic nitpick; it's easier to read if the variables are declared as close as possible to where they are used.